### ISSUE TITLE
claat drops text from hyperlink when exporting to markdown

### ISSUE THREAD

**Issue by petele** [googlecodelabs/tools#15]
When exporting to markdown, claat seems to drop the link text for some types of links. It appears to work fine when exporting as HTML.

From what I can tell, it happens when the hyperlink in the doc is meant to be a button (has a green background in docs).

Repro Steps:
1. Export Your First PWA Code Lab (1rpHleSSeY-MJZ8JvncvYA8CFqlnlcrW8-a4uEaqizPY) as markdown.

Expected result:
No links in the form of `[](https://github.com/googlecodelabs/your-first-pwapp/archive/master.zip)`

**Comment by x1ddos** [comment]
Yeah, it's a known issue. I can't recall why I didn't add the text at the moment but there was a reason.
Anyway, thanks for reminding. I'll fix it.

**Comment by gauntface** [comment]
Just hit this with the push codelab where the link text was dropped and replaced with the text 'Link'.

**Comment by x1ddos** [comment]
Will be working on this tomorrow.
